### ISSUE TITLE
PCC Drop in timm_vgg19_bn, PCC=0.9569443831910941

### ISSUE THREAD

**Issue by kamalrajkannan78** [tenstorrent/tt-xla#2803]
### Describe the bug 

- timm_vgg19_bn have `PCC=0.9569443831910941` [jan12_timm_vgg19_bn.log](https://github.com/user-attachments/files/24569425/jan12_timm_vgg19_bn.log)

### Root cause analysis

- Pcc drop comes from [features block](https://github.com/huggingface/pytorch-image-models/blob/78c3724f4545c0067d0b4d2f7df6337196be01ce/timm/models/vgg.py#L230). 
- Features block has 53 layers. Here are the pcc after each layer - [jan12_timm_vgg19_bn_d8.log](https://github.com/user-attachments/files/24569200/jan12_timm_vgg19_bn_d8.log), [loader.py](https://github.com/user-attachments/files/24569201/loader.py) , [full_model_arch.txt](https://github.com/user-attachments/files/24569342/arch.txt)
 
 - From commented pcc values in loader.py , it's clear that first time pcc become <0.99 after layer31 but it becomes >0.99 after layer32. So it's not origin.
 - Second time pcc becomes <0.99 after layer34 and it never become >0.99 till end. so this is the origin of drop and it is batchnorm op
 - But bacthnorm sanity passed with valid pcc even with saved inputs - [jan12_timm_vgg19_bn_s.log](https://github.com/user-attachments/files/24569272/jan12_timm_vgg19_bn_bn_s.log), [loader.py](https://github.com/user-attachments/files/24569273/loader.py)
 - Expanded sanity by adding preceding conv op too. It also passed. - [jan12_timm_vgg19_bn_conv_s.log](https://github.com/user-attachments/files/24569291/jan12_timm_vgg19_bn_conv_s.log), [loader.py](https://github.com/user-attachments/files/24569290/loader.py)

### Repro

```
git checkout kkannan/jan12_vgg_pcc_drop_debug
git submodule update --init —recursive
git lfs pull --include ‘*.pt'

# model cases 
pytest tests/runner/test_models.py::test_all_models_torch[vgg/pytorch-timm_vgg19_bn-single_device-inference] -svv

# to run the mentioned cases, place the contents of attached scripts in corresponding original scripts and rerun above command
```
